src/main/java/io/vertx/core/net/package-info.java

```diff
@@ -290,6 +290,40 @@
  *
  * By default, multiple connection attempts are disabled.
  *
+ * [[logging_network_activity]]
+ * === Logging network activity
+ *
+ * For debugging purposes, network activity can be logged:
+ *
+ * [source,$lang]
+ * ----
+ * {@link examples.NetExamples#exampleNetworkActivityLoggingOnServer}
+ * ----
+ *
+ * for the client
+ *
+ * [source,$lang]
+ * ----
+ * {@link examples.NetExamples#exampleNetworkActivityLoggingOnClient}
+ * ----
+ *
+ * Network activity is logged by Netty with the `DEBUG` level and with the `io.netty.handler.logging.LoggingHandler`
+ * name. Netty will try to locate the following logger implementations, in the following order:
+ *
+ * - Slf4j
+ * - Log4j
+ * - JDK
+ *
+ * The presense of the slf4j or log4j classes on the classpath is enough to pick up the logging implementation.
```

Comment:
Add a sentence that this resolution is independent from the vert.x log delegate (that can be very confusing actually).